apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifi_manager_main.c

```diff
@@ -805,10 +1078,9 @@ int wifi_manager_itc(int argc, FAR char *argv[])
 	itc_wifi_manager_set_mode_p();
 
 	itc_wifi_manager_get_mode_p(); // set softap mode inside this function
-
 	itc_wifi_manager_connect_disconnect_ap_p(); // change to station mode and try to connect to ap
 
-	itc_wifi_manager_scan_ap_p(); // Reinitialized wifi manager with the callback hander for scan results
+	//itc_wifi_manager_scan_ap_p(); // Reinitialized wifi manager with the callback hander for scan results
```

Comment:
Hi @manoj11588612,
The reason for not doing the scanning here is probably the state error in the WiFi Manager.
Please add WIFITEST_WAIT right after calling wifi_manager_scan_ap(),
and then, add WIFITEST_SIGNAL in wifi_scan_ap_done_cb() callback function.
(WIFITEST_SIGNAL should be called within wifi_scan_ap_done_cb() whether it succeeded or not.)
Then, the scanning operation does not need to be done last in wifi_manager_itc(),
and the reconnect operation will also be successful.

Reply:
I agree with your comment.
I added your comment and checked . ALL testcase gettign passed.
Please review.